src/software/ai/hl/stp/play/play.cpp

```diff
@@ -403,6 +419,59 @@ Play::assignTactics(const WorldPtr &world_ptr, TacticVector tactic_vector,
         current_tactic_robot_id_assignment};
 }
 
+std::vector<Play::RobotSubstituionParam> Play::trySubstituteInjuredRobots(const WorldPtr& world_ptr){
+    // Don't do anything if there aren't any injured robots  
+    const std::vector<Robot>& injured_robots = world_ptr->friendlyTeam().getInjuredRobots();
+    if(injured_robots.size()  == 0)
```

Comment:
using explicit `{ }` is a part of our code style guide: https://github.com/UBC-Thunderbots/Software/blob/bd3317f338c7086fdb9dd05c0db8a11e601efe26/docs/code-style-guide.md?plain=1#L307